modules/data.land/NAMESPACE

```diff
@@ -58,4 +58,3 @@ export(write_veg)
 importFrom(magrittr,"%>%")
 importFrom(ncdf4,ncvar_get)
 importFrom(rlang,"%||%")
-importFrom(rlang,.data)
```

Comment:
This looks like an out-of-date Roxygen output -- there are definitely files in data.land that use `.data` and aren't edited in this PR, so this importFrom should still be present.

Reply:
Yep this must've been edited by mistake.